scripts/docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ MAINTAINER Devis Lucato (https://github.com/dluc)
 
 LABEL Tags="Azure,IoT,Solutions,React,SPA"
 
+ARG user=app
+
+RUN addgroup $user && adduser -D -G $user $user
```

Comment:
can you sync this with the line of the other PR?

Reply:
the command 'groupadd' and 'useradd' are not supported by /bin/sh in Busybox. They are supported by bash shell. I have to change them to 'addgroup' and 'adduser' which are supported by /bin/sh.

Reply:
the command 'groupadd' and 'useradd' are not supported by /bin/sh in Busybox. They are supported by bash shell. I have to change them to 'addgroup' and 'adduser' which are supported by /bin/sh.